Assets/Scripts/Scenes/Lobby/RoomSettingPanel/LookDisplayObject.cs

```diff
@@ -0,0 +1,128 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Cysharp.Threading.Tasks;
+using Fusion;
+using Common.Network;
+
+namespace Scenes.Lobby.RoomSettingPanel
+{
+    public class LookDisplayObject : MonoBehaviour
+    {
+        [SerializeField] private Camera _displayCamera;
+        [SerializeField] private GameObject�@_uiPanel;
+        [SerializeField] private NetworkRunner _runnerPrefab;
+        [SerializeField] private float _motionTime = 1f;
+
+        private int _isOpened = 0; //�f�B�X�v���CUI�̕\�����
```

Comment:
```suggestion
        private DisplayState _isOpened = DisplayState.close; //ディスプレイUIの表示状態
```
```c#
public enum DisplayState
{
close,
opening,
opened
}
``` 
複数状態を管理する場合は、enumにして掻き出した方が、関数単体で見たときに、その値が何を意味するかが分かりやすいので、enumにしておいて欲しいです（enum内の命名に関しては例なので任せます）